### ISSUE TITLE
Default styles fail accessibility contrast lighthouse rules

### ISSUE THREAD

**Issue by NateRadebaugh** [siddharthkp/react-ui#30]
Out of the box (light theme), running Lighthouse audits, I'm not getting 100% in accessibility...

> "Background and foreground colors do not have a sufficient contrast ratio."

![image](https://user-images.githubusercontent.com/130445/79073774-f9c75c80-7cad-11ea-97ff-e89042352249.png)

**Comment by siddharthkp** [comment]
That's a great bug report! I'll look into it 👍

**Comment by ankibalyan** [comment]
Can we integrate some tool to check color contrasts during build or after build?

Something on these lines.

https://github.com/bbc/color-contrast-checker
https://dev.to/alvaromontoro/building-your-own-color-contrast-checker-4j7o

**Comment by siddharthkp** [comment]
> Can we integrate some tool to check color contrasts during build or after build?

When it comes to default themes, I wouldn't try to solve it with CI/build tooling. Assigned it to myself to work on the default colors
